src/monitoring_service/handlers.py

```diff
@@ -371,13 +372,16 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
             ).transact({'from': context.ms_state.address})
 
             log.info(
-                'Calling `monitor` on channel',
+                'Sent transaction calling `monitor` for channel',
                 token_network_address=channel.token_network_address,
                 channel_identifier=channel.identifier,
                 transaction_hash=encode_hex(tx_hash),
             )
             assert tx_hash is not None
 
+            # Add tx hash to list of waiting transactions
+            context.waiting_transactions.append(encode_hex(tx_hash))
```

Comment:
Isn't this already hex encoded?

Reply:
No, it's `HexBytes`. That should be fine as well but a string feels cleaner.